src/python/pants/jvm/subsystems.py

```diff
@@ -78,6 +78,21 @@ class EnvironmentAware:
             """
         ),
     )
+    enable_lockfile_targets = BoolOption(
+        default=True,
+        help=softwrap(
+            """
+            Create targets for all JVM lockfiles defined in `[jvm].resolves`.
+
+            The lockfile targets will then be used as dependencies to the targets that use them,
+            invalidating source targets per resolve when the lockfile changes.
+
+            If another targets address is in conflict with the created lockfile target, it will
+            shadow the lockfile target and it will not be available as a dependency.
+            """
```

Comment:
This help text is very vague only because I have no idea what targets are involved etc. ;) Or even if it's correct :P (well technically it'll be un-true until there are targets that will add a dependency to the synthetic lockfile targets..)